module-2/backend/week_four.md

```diff
@@ -10,14 +10,26 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 * What is a cookie?
+    * A Cookie is a data structure that is sent back and forth between client and server
 * What’s the difference between a session and a cookie?
+    * A session tracks a clients data from the server side where a cookie is exchanged between client and server
 * What’s a flash and when do you want to use flashes?
+    * Flashes notify the client when something needs to be communicated from the server
 * Why do people say “HTTP is stateless”?
+    * HTTP is stateless because it doesn't track any previous requests. It only sends and receives as it were the first time
```

Comment:
Because a stateless protocol does not require the server to retain session information or status about each communications partner for the duration of multiple requests. HTTP is a stateless protocol, which means that the connection between the browser and the server is lost once the transaction ends